automation/src/test/scala/org/broadinstitute/dsde/workbench/WebBrowserSpec.scala

```diff
@@ -57,10 +78,10 @@ trait WebBrowserSpec extends WebBrowserUtil with ExceptionHandling with LazyLogg
     }
   }
 
-  private def runHeadless(testCode: (WebDriver) => Any) = {
-    val defaultChrome = FireCloudConfig.ChromeSettings.chromedriverHost
-    implicit val driver = new RemoteWebDriver(new URL(defaultChrome), DesiredCapabilities.chrome())
-    driver.manage.window.setSize(new org.openqa.selenium.Dimension(1600, 2400))
+  private def runHeadless(capabilities: DesiredCapabilities, testCode: (WebDriver) => Any): Unit = {
```

Comment:
Not super important but are the tests able to run headlessly now?

Reply:
not sure!

Reply:
Possibly after #67 - will check after rebasing onto that